packages/x/xcb-util-errors/package.yml

```diff
@@ -0,0 +1,20 @@
+name       : xcb-util-errors
+version    : 1.0.1
+release    : 1
+source     :
+    - https://xcb.freedesktop.org/dist/xcb-util-errors-1.0.1.tar.gz : cfbd3b022bdb27a6921a4abd6b41f4071b4e4960447598abd30955d3454f4d99
```

Comment:
Can you please use `https://www.x.org/releases/individual/xcb/xcb-util-errors-1.0.1.tar.gz` as the source instead?